app/src/components/pomodoro/circular-progress.tsx

```diff
@@ -33,7 +33,7 @@ export const ProgressCircle = ({ colour, percentage }: ProgressCircleProps): Rea
       // cy={-105} // Can remove this and inclde in transform translate attribute
```

Comment:
For improved readability we can remove `transform` and and the translated y-coordinate to the ` cy` attribute.

```
<circle
      data-testid={`progress-circle-${colour}-test`}
      r={r}
      cx={X_AXIS}
      cy={205} // replace the a const like Y_AXIS
      fill="transparent"
      stroke={stroke} // remove colour as 0% sets full circumference
      strokeWidth={12}
      strokeDasharray={circ}
      strokeDashoffset={strokeDashoffset}
      strokeLinecap="round"
    ></circle>
```